Dockerfile.rocm

```diff
@@ -39,79 +41,150 @@ RUN apt-get update && apt-get install -y \
     build-essential \
     wget \
     unzip \
-    nvidia-cuda-toolkit \
     tmux \
     ccache \
  && rm -rf /var/lib/apt/lists/*
 
-### Mount Point ###
-# When launching the container, mount the code directory to /app
+# When launching the container, mount the code directory to /vllm-workspace
 ARG APP_MOUNT=/vllm-workspace
-VOLUME [ ${APP_MOUNT} ]
 WORKDIR ${APP_MOUNT}
 
-RUN python3 -m pip install --upgrade pip
-RUN python3 -m pip install --no-cache-dir fastapi ninja tokenizers pandas
+RUN pip install --upgrade pip
+RUN pip install --no-cache-dir fastapi ninja tokenizers pandas
```

Comment:
QQ: Why do we have to install those dependencies separately? Aren't they installed while installing `torch` or installing `requirements-common.txt`?

Reply:
They were in the previous Dockerfile.rocm (current upstream Dockerfile.rocm) but no longer necessary, and have been removed.